test_analyze.py

```
@@ -0,0 +1,2 @@
+import os
+os.system('flutter analyze')
```

Ruff (0.15.5)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)